src/Microsoft.SqlTools.ServiceLayer/ObjectManagement/ObjectTypeHandler.cs

```diff
@@ -103,6 +104,27 @@ public virtual Task Drop(string connectionUri, string objectUrn, bool throwIfNot
             return Task.CompletedTask;
         }
 
+        public virtual Task CreateCredential(string connectionUri, string objectUrn, bool throwIfNotExist)
```

Comment:
It's unclear how a credential is being created here. Could you add a comment providing more details?

Reply:
Oh sorry, I forgot to remove this method when cleaning the code